lib/ably/realtime/channel.rb

```diff
@@ -179,15 +184,26 @@ def presence
         @presence
       end
 
-      # Return the message history of the channel.
+      # Return the message history of the channel
+      #
+      # Once attached to a channel, you can retrieve messages published on the channel before the
```

Comment:
> The comments say "Once attached to a channel, you can retrieve messages published on the channel before the channel was attached with the option until_attach: true. This is very useful for developers who wish to subscribe to new realtime messages yet also display historical messages with the guarantee that no messages have been missed."

I know. I find it lacking. "Once attached to a channel" doesn't tell the user that he __needs__ to attach this channel to use this feature.

The documentation tells that it is "uncommon" to call `attach` since it is implicit with `publish` and `subscribe`: https://github.com/ably/ably-ruby/blob/4425f4e3fba3df9a2ac7cdd23c0fe5cd6945222f/lib/ably/realtime/channel.rb#L154-L155.

`publish` and `subscribe` do not mention `attach` at all: https://github.com/ably/ably-ruby/blob/4425f4e3fba3df9a2ac7cdd23c0fe5cd6945222f/lib/ably/realtime/channel.rb#L103, https://github.com/ably/ably-ruby/blob/4425f4e3fba3df9a2ac7cdd23c0fe5cd6945222f/lib/ably/realtime/channel.rb#L130.

Even `presence` does an implicit `attach` https://github.com/ably/ably-ruby/blob/4425f4e3fba3df9a2ac7cdd23c0fe5cd6945222f/lib/ably/realtime/channel.rb#L183.

Actually `history` is the only one not doing an implicit `attach` on the public method of this class (apart from `unsubscribe` for obvious reason I guess). And it is on the other hand the only one mentioning any relation to `attach` on a specific option.

I'm also wondering what happens in `unsubscribe` and if it auto-detach from the channel too (which would sound reasonable). https://github.com/ably/ably-ruby/blob/4425f4e3fba3df9a2ac7cdd23c0fe5cd6945222f/lib/ably/realtime/channel.rb#L142-L151